test/support/helper.ex

```diff
@@ -8,12 +8,32 @@ defmodule EventBus.Support.Helper do
 
     require Logger
 
+    use GenServer
```

Comment:
This doesn't have to be a GenServer; I prefer to leave it as is to provide example without GenServer

Reply:
Okay, I'll just revert the `GenServer`s since I need them all to have my test pass consistently. If it passes in `travis` then it might be my machine.